.gitignore

```diff
@@ -68,3 +68,4 @@ build
 bin
 out
 test-results
+docs/airtable-to-markdown.py
```

Comment:
I think we shall put everything into this one repo here.

Reply:
Apologies, I didn't quite understand your statement. Could you please clarify what you are referring to?